projects/docs/src/assets/scully-routes.json

```diff
@@ -0,0 +1,33 @@
+[
+    { "route": "/" },
+    { "route": "/docs" },
+    { "route": "/docs/getting-started" },
+    { "route": "/docs/accordion" },
+    { "route": "/docs/breadcrumb" },
+    { "route": "/docs/button" },
+    { "route": "/docs/checkbox" },
+    { "route": "/docs/chip" },
+    { "route": "/docs/collapse" },
+    { "route": "/docs/datepicker" },
+    { "route": "/docs/dropdown" },
+    { "route": "/docs/loader" },
+    { "route": "/docs/modal" },
+    { "route": "/docs/notification" },
+    { "route": "/docs/pagination" },
+    { "route": "/docs/progressbar" },
+    { "route": "/docs/radiogroup" },
+    { "route": "/docs/rating" },
+    { "route": "/docs/slider" },
+    { "route": "/docs/stepper" },
+    { "route": "/docs/table" },
+    { "route": "/docs/tabs" },
+    { "route": "/docs/textarea" },
+    { "route": "/docs/textboxgroup" },
+    { "route": "/docs/textlabel" },
+    { "route": "/docs/timeline" },
+    { "route": "/docs/timer" },
+    { "route": "/docs/toggle" },
+    { "route": "/docs/tooltip" },
+    { "route": "/docs/video" },
+    { "route": "/404" }
+]
```

Comment:
I ran the `build:docs` on my machine, and this file seems to be generated every single time. Is this supposed to be in the `assets` folder? Shouldn't this file get dropped in the `dist` folder instead?

If it is supposed to be dropped here, can you at least run a prettier on the file after it's generated? because the generated file is minified and it will always show git changes when running a build.

Reply:
it will be generated in assets folder first then copied to dist folder. default scully function

Reply:
then you can add a `post` npm hook to delete the folder
```bash
rm projects/docs/src/assets/scully-routes.json
```

or you can add it to `.gitignore`